stdlib/test/os/test_atomic.mojo

```diff
@@ -52,7 +52,7 @@ fn test_atomic():
 
 
 # CHECK-LABEL: test_atomic_floating_point
-fn test_atomic_floating_poInt__():
+fn test_atomic_floating_point__():
```

Comment:
**Suggestion** Pre-existing, but do you mind dropping (removing) the two trailing underscores in this function name while you're here?

Reply:
Sure. Done